pyproject.toml

```diff
@@ -53,7 +53,7 @@ implicit_reexport = true
 allow_untyped_calls = true
 
 [tool.poe.tasks]
-test = "pytest"
+test = { cmd = "pytest", env = { YAGNA_SUBNET = "public" } }
```

Comment:
Why this change is required as there is no tests introduced in this PR?

Reply:
Old test were failing as there are no providers on default subnet in yappai version used by golem-core